modules/ossm-installing-cert-manager.adoc

```diff
@@ -25,10 +25,29 @@ You must install cert-manager before installing your `Istio` resource.
 $ oc create namespace istio-system
 ----
 
-. Create the root issuer by creating an `Issuer` object in a YAML file.
+. Patch the cert-manager Operator to install the `istio-csr` agent by running the following command:
++
+[source, terminal]
+----
+$ oc -n cert-manager-operator patch subscription openshift-cert-manager-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"UNSUPPORTED_ADDON_FEATURES","value":"IstioCSR=true"}]}}}'
```

Comment:
Because this command is quite long, consider reformatting it, as shown in the OCP docs guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#backslash-for-long-lines-of-code